### PR TITLE
Fix properties still using Hibernate ORM 6.4 / Hibernate Search 7.0  in POMs

### PR DESCRIPTION
**Pull request by yrodiere**
I got it wrong, and I predict I'll get it wrong again...

### CODE REVIEW

File: docs/pom.xml [hunk 1]
```diff
@@ -39,8 +39,8 @@
         <vale.image>docker.io/jdkato/vale:v2.15.5</vale.image>
 
         <!-- Keep these in sync with the full versions in bom/application/pom.xml -->
-        <hibernate-orm.version-for-documentation>6.4</hibernate-orm.version-for-documentation>
-        <hibernate-search.version-for-documentation>7.0</hibernate-search.version-for-documentation>
+        <hibernate-orm.version-for-documentation>6.5</hibernate-orm.version-for-documentation>
+        <hibernate-search.version-for-documentation>7.1</hibernate-search.version-for-documentation>
```
Comment (gsmet) [inline docs/pom.xml:43]: Couldn't we use the Maven plugin we use to split the versions?

**Review by gsmet — APPROVED** [review]

File: docs/pom.xml (hunk 1 above)
Comment (gsmet) [reply docs/pom.xml:43]: BTW, I don't mind keeping it that way.

Comment (yrodiere) [reply docs/pom.xml:43]: I'll look into this, but in a separate PR.

Comment (yrodiere) [reply docs/pom.xml:43]: https://github.com/quarkusio/quarkus/pull/40964

**Pull request merged**